styles/component-styles/alert.css

```diff
@@ -0,0 +1,34 @@
+.alert-container{
+    padding: 2rem;
+    display: flex;
+    flex-direction: column;
+    row-gap: 2rem;
+}
+
+.error{
+    background-color: #ff4040;
+    padding: 1rem;
+    border-radius: 1rem;
+    border: 2px solid darkred;
+}
+
+.warning{
+    background-color: #fff44f;
```

Comment:
Use colors in variables. It will help you to reuse

Reply:
Oh! yes, I forgot to use variables. Thanks for pointing it out. rectified!